src/components/ListItem.jsx

```diff
@@ -1,28 +1,44 @@
 import './ListItem.css';
 import { updateItem, deleteItem } from '../api/firebase.js';
-import { useState, useEffect, useMemo } from 'react';
+import { useState, useEffect } from 'react';
 import { ONE_DAY_IN_MILLISECONDS } from '../utils/dates.js';
 import { purchaseUrgency } from '../utils/hooks.js';
 
 export function ListItem({ listPath, item }) {
-	/* Returns a boolean that is passed into isChecked useState
-	On render, box is checked if purchased less than a day ago */
+	//Time variables for each item on render
+	const lastPurchase =
```

Comment:
<img src="https://media2.giphy.com/media/TSXSPZUSW0Lr9mYm8h/giphy.gif"/>

The screen captures really helped explain it quickly. Neat to see how most of the code I had refactored could be re-refactored into an even better useEffect! It may be an edge case, but it's a situation I could easily see happening. Suggestions have been incorporated!